Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM haproxy:2.0-alpine
+
```

Comment:
we should maybe set a maintainer in the dockerfile. Please see https://docs.docker.com/engine/reference/builder/#maintainer-deprecated